src/main/kotlin/no/nav/syfo/infrastructure/kafka/esyfovarsel/ArbeidstakervarselProducer.kt

```diff
@@ -3,21 +3,33 @@ package no.nav.syfo.infrastructure.kafka.esyfovarsel
 import no.nav.syfo.application.IVarselProducer
 import no.nav.syfo.domain.PersonIdent
 import no.nav.syfo.domain.Varsel
+import no.nav.syfo.infrastructure.kafka.esyfovarsel.dto.*
 import java.util.*
 import org.apache.kafka.clients.producer.KafkaProducer
 import org.apache.kafka.clients.producer.ProducerRecord
 import org.slf4j.LoggerFactory
 
-// TODO: Add esyfovarsel-hendelse as kafkaproducer value
-class ArbeidstakervarselProducer(private val kafkaArbeidstakervarselProducer: KafkaProducer<String, String>) : IVarselProducer {
+class ArbeidstakervarselProducer(private val kafkaArbeidstakervarselProducer: KafkaProducer<String, EsyfovarselHendelse>) : IVarselProducer {
```

Comment:
Bør vi spesifisere at det gjelder forhåndsvarsel? 😊
```suggestion
class ArbeidstakerForhandsvarselProducer(private val kafkaArbeidstakerForhandsvarselProducer: KafkaProducer<String, EsyfovarselHendelse>) : IVarselProducer {
```

Reply:
Ja, det kan være greit. I hvert fall så lenge HendelseType vi angir gjelder spesifikt forhåndsvarsel.

Reply:
`HendelseType` kan vi også endre hvis vi tenker at vi vil lage en generell? 😊 Men det er kanskje greit å ikke optimalisere for hva som muligens kommer i fremtiden

Reply:
Ja, vi kan godt la den være spesifikk for nå.